src/app.js

```diff
@@ -39,6 +39,8 @@ F.manager.strategy['new-if-missing'] = require('./managers/run-strategies/new-if
 F.manager.strategy['new-if-persisted'] = require('./managers/run-strategies/new-if-persisted-strategy');
 F.manager.strategy['new-if-initialized'] = require('./managers/run-strategies/new-if-initialized-strategy');
 
+F.manager.AuthManager = require('./managers/auth-manager');
```

Comment:
I named it this way to be consistent with the other managers, i.e.
```javascript
F.manager.ScenarioManager = require('./managers/scenario-manager');
F.manager.RunManager = require('./managers/run-manager');
```

I suggest to change all manager names if we want to keep this consistent